runtime/compiler/codegen/J9RecognizedMethodsEnum.hpp

```diff
@@ -1149,6 +1149,65 @@
    java_lang_invoke_MethodHandleImpl_profileBoolean,
    java_lang_invoke_MethodHandleImpl_isCompileConstant,
 
+   java_lang_invoke_VarHandleBooleans_Array_method,
```

Comment:
The sheer number of these makes things pretty tedious to read IMO. Since each represents many methods anyway, do you think it would make sense to group them up further? It looks like we only need to distinguish instance fields vs. static fields vs. array elements (including in byte array views). That could eliminate a lot of the repetition in recognizing the class names as well considering how systematic the naming is

Reply:
I agree. Based on your suggestion I have grouped them in a way that allows us to distinguish them based on only the info that's relevant for handling them.